.circleci/config.yml

```diff
@@ -1,46 +1,34 @@
 version: 2.1
 orbs:
-  singularity: singularity/singularity@1.0.10
   python: circleci/python@0.3
 jobs:
-  singularity-image:
-    executor: singularity/machine
+  docker:
+    machine: true
     steps:
       - checkout
-      - singularity/install-go:
-          go-version: '1.13'
-      - singularity/debian-install-3:
-          singularity-version: 3.5.0
       - run:
-          # copied from singularity/build-image to add branch-specific build
-          name: Build Singularity Container
+          name: Build docker image
           command: |
-            NOSUDO=false
-            which sudo || NOSUDO=true
-            if [[ "$NOSUDO" == "false" ]]; then
-                if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
-            fi
-            if [ -f "$BASH_ENV" ]; then source $BASH_ENV; fi
-            $SUDO SINGULARITYENV_OPHYS_ETL_TAG=${CIRCLE_BRANCH} singularity build suite2p_container.img src/ophys_etl/transforms/suite2p_container.def
+            docker build --build-arg OPHYS_ETL_TAG=${CIRCLE_BRANCH} -t alleninstitutepika/ophys_etl_pipelines:${CIRCLE_SHA1} docker/
       - run:
-          name: Run Singularity Container pytest
+          name: Run docker image pytest
```

Comment:
since our use case is using singularity to run a docker image, do you think it is worth testing against that? I don't have a sense of how much we can rely on that working,

Reply:
I guess we're not doing that in ophys_nway and haven't seen trouble.

Reply:
we decided this is out-of-scope, but we noted it for future consideration